spaceship/spec/UI/select_team_spec.rb

```diff
@@ -23,13 +23,13 @@
 
         it "Lets the user select the team if in multiple teams" do
           allow($stdin).to receive(:gets).and_return("2")
-          expect(subject.select_team).to eq("SecondTeam") # a different team
+          expect(subject.select_team).to eq("XXXXXXXXXX") # a different team
```

Comment:
This kind of makes the test less useful, could you add a second test that selects `"1"` and therefore selects `SecondTeam`